hibernate-orm-panache-quickstart/src/main/resources/application.properties

```diff
@@ -1,10 +1,56 @@
-%prod.quarkus.datasource.db-kind=postgresql
-%prod.quarkus.datasource.username=quarkus_test
-%prod.quarkus.datasource.password=quarkus_test
-%prod.quarkus.datasource.jdbc.url=jdbc:postgresql://localhost/quarkus_test
-%prod.quarkus.datasource.jdbc.max-size=8
-%prod.quarkus.datasource.jdbc.min-size=2
+quarkus.package.type=uber-jar
 
 quarkus.hibernate-orm.database.generation=drop-and-create
+quarkus.datasource.db-kind=postgresql
+quarkus.datasource.jdbc.max-size=8
+quarkus.datasource.jdbc.min-size=2
 quarkus.hibernate-orm.log.sql=true
 quarkus.hibernate-orm.sql-load-script=import.sql
+quarkus.datasource.jdbc.acquisition-timeout = 10
+
+%dev.quarkus.datasource.username=${AZURE_CLIENT_NAME}@${DBHOST}
+%dev.quarkus.datasource.jdbc.url=jdbc:postgresql://${DBHOST}.postgres.database.azure.com:5432/${DBNAME}?\
+  authenticationPluginClassName=com.azure.identity.providers.postgresql.AzureIdentityPostgresqlAuthenticationPlugin\
+  &sslmode=require\
+  &azure.clientId=${AZURE_CLIENT_ID}\
+  &azure.clientSecret=${AZURE_CLIENT_SECRET}\
+  &azure.tenantId=${AZURE_TENANT_ID}
+
+%prod.quarkus.datasource.username=${AZURE_MI_NAME}@${DBHOST}
+%prod.quarkus.datasource.jdbc.url=jdbc:postgresql://${DBHOST}.postgres.database.azure.com:5432/${DBNAME}?\
```

Comment:
If user uses Service Connector with --client-type java, then the prop could be
%prod.quarkus.datasource.jdbc.url=${AZURE_MYSQL_CONNECTIONSTRING}&authenticationPluginClassName=com.azure.identity.providers.postgresql.AzureIdentityPostgresqlAuthenticationPlugin